src/plugins/management/public/plugin.ts

```diff
@@ -118,6 +123,61 @@ export class ManagementPlugin implements Plugin<ManagementSetup, ManagementStart
       }
     );
 
+    const getNavLinksByNavGroupId = async (navGroupId: string) => {
+      const [coreStart] = await core.getStartServices();
+      const navGroupMap = await coreStart.chrome.navGroup
+        .getNavGroupsMap$()
+        .pipe(first())
+        .toPromise();
+      const navLinks = navGroupMap[navGroupId]?.navLinks;
+      return getSortedNavLinks(
+        fulfillRegistrationLinksToChromeNavLinks(
+          navLinks || [],
+          coreStart.chrome.navLinks.getAll()
+        ).filter((item) => !item.hidden),
+        (currentItem, parentItem) => {
+          /**
+           * Hide sub items in landing page
+           */
+          if (
+            currentItem.itemType === LinkItemType.LINK &&
+            parentItem?.itemType === LinkItemType.PARENT_LINK
+          ) {
```

Comment:
Q: how do we know this is landing page?

Reply:
The comment may lead to some misunderstanding. Actually it is about we need to hide all the sub items in landing page, and this function will only be called by landing pages to collect all the nav links that await to display.

Reply:
### Before:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/44e96663-ad8f-4ab0-820f-726813961499">

### After:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/5002a56d-c809-47a3-a9f0-511df3e1445a">

Reply:
### Before:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/44e96663-ad8f-4ab0-820f-726813961499">

### After:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/5002a56d-c809-47a3-a9f0-511df3e1445a">